### ISSUE TITLE
pretained ckpts?

### ISSUE THREAD

**Issue by zezeze97** [foranonymous123/CNNforFaultInterpretation#9]
Hello, your work on fault recongnition is excellent and has been very inspiring for my research. Could you please provide the pre-trained weights of your model or the inference results on the Thebe test set? It would be convenient for us to compare with our method.
